src/quo2/components/wallet/account_card/view.cljs

```diff
@@ -0,0 +1,105 @@
+(ns quo2.components.wallet.account-card.view
+  (:require [react-native.core :as rn]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.icon :as icon]
+            [quo2.components.wallet.account-card.style :as style]
+            [quo2.components.buttons.button.view :as button]
+            [utils.i18n :as i18n]
+            [quo2.components.markdown.text :as text]
+            [quo2.theme :as theme]))
+
+(defn loading-view
+  [{:keys [customization-color type theme metrics?]}]
+  (let [watch-only? (= :watch-only type)
+        empty?      (= :empty type)]
+    [rn/view (style/card customization-color watch-only? metrics? theme)
+     [rn/view {:style {:flex-direction :row :align-items :center}}
+      [rn/view {:margin-right 8 :margin-top 2 :style (style/loader-view 16 16 watch-only? theme)}]
+      [rn/view {:style style/watch-only-container}
+       [rn/view {:style (style/loader-view 57 8 watch-only? theme)}]
+       (when watch-only? [icon/icon :reveal {:color colors/neutral-50 :size 12}])]]
+     [rn/view {:margin-top 13 :style (style/loader-view (if empty? 56 80) 16 watch-only? theme)}]
+     (when metrics?
+       [rn/view
+        {:margin-top 10
+         :style      (style/loader-view (if empty? 37 96) 8 watch-only? theme)}])]))
+
+(defn user-account
+  [{:keys [state name balance percentage-value loading? amount customization-color type emoji metrics?
+           theme]}]
+  (let [watch-only?        (= :watch-only type)
+        empty?             (= :empty type)
+        account-amount     (if (= :empty state) "€0.00" amount)
+        account-name       (if (= :empty state) "Account 1" name)
+        account-percentage (if (= :empty state) "€0.00" percentage-value)
+       ]
```

Comment:
Dangling square bracket. Reminder that dangling parentheses won't be fixed by the formatter.